Katello/hammer-cli-katello
# Fixes #36742 - Add other content types to cv info command

What are the changes introduced in this pull request?
* Added new file types to loop through in the Repositories collections
* Added some `hide_blank: true, hide_empty: true` to the collections and ak/components so if they don't have those content types then don't show them
* Added `Fields::Boolean` to `solve_deps` and `composite` to make sure they come through correctly.
* Added rubocop disable line length because my hide attributes were pushing the line over one hundred by 1-5

What are the testing steps for this pull request?
* Check out PR
* Create repos for Ansible Collections, Deb, File, Python, yum, OSTree, and Docker and synchronize them
* Create a CV with all of those and publish them
* Run `hammer content-view info --id X` and see if the results show up with all the added content types:

Before my PR:

```bash
[vagrant@hammer hammer-cli-katello]$ hammer content-view info --id 2
WARNING: File locale/bn/LC_MESSAGES/hammer-cli-katello.mo outdated, regenerate with 'make all-mo'
WARNING: File locale/gu/LC_MESSAGES/hammer-cli-katello.mo outdated, regenerate with 'make all-mo'
WARNING: File locale/pt_BR/LC_MESSAGES/hammer-cli-katello.mo outdated, regenerate with 'make all-mo'
Id:                           2
Name:                         All
Label:                        All
Composite:                    false
Description:
Content Host Count:           0
Solve Dependencies:           false
Organization:                 Default Organization
Yum Repositories:
 1) Id:    2
    Name:  Modules
    Label: Modules
 2) Id:    1
    Name:  Animals
    Label: Animals
Container Image Repositories:
 1) Id:    5
    Name:  Docker
    Label: Docker
OSTree Repositories:
 1) Id:    6
    Name:  OSTree
    Label: OSTree
Lifecycle Environments:
 1) Id:   1
    Name: Library
Versions:
 1) Id:        2
    Version:   1.0
    Published: 2024/02/01 15:26:29
Components:

Activation Keys:
```

With my PR:
```bash
[vagrant@hammer hammer-cli-katello]$ hammer content-view info --id 2
Id:                           2
Name:                         All
Label:                        All
Composite:                    no
Description:
Content Host Count:           0
Solve Dependencies:           no
Organization:                 Default Organization
Yum Repositories:
 1) Id:    2
    Name:  Modules
    Label: Modules
 2) Id:    1
    Name:  Animals
    Label: Animals
File Repositories:
 1) Id:    3
    Name:  File
    Label: File
Container Image Repositories:
 1) Id:    5
    Name:  Docker
    Label: Docker
OSTree Repositories:
 1) Id:    6
    Name:  OSTree
    Label: OSTree
Ansible Repositories:
 1) Id:    7
    Name:  Ansible
    Label: Ansible
File Repositories:
 1) Id:    3
    Name:  File
    Label: File
Debian Repositories:
 1) Id:    8
    Name:  Debian
    Label: Debian
Python Repositories:
 1) Id:    4
    Name:  Python
    Label: Python
Lifecycle Environments:
 1) Id:   1
    Name: Library
Versions:
 1) Id:        2
    Version:   1.0
    Published: 2024/02/01 15:26:29
```
